product_lot_sequence/readme/newsfragments/3037.feature.rst

```diff
@@ -0,0 +1 @@
+Add a field "default lot sequence" in settings > inventory
```

Comment:
**chore**: I don't think newsfragments are used in OCA anymore

Reply:
> **chore**: I don't think newsfragments are used in OCA anymore

hi @aleuffre just wondering what is this statement based on? Because they are still in the [official OCA module template](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme/newsfragments).
And if they are really not used any longer what should be used instead?

Reply:
Hi @ivs-cetmix , this is an interesting question but not really relevant to this PR. 

Why don't you open an issue in https://github.com/OCA/maintainer-tools to discuss whether fragments are still relevant or not?

Reply:
@francesco-ooops just to clarify:
- Request to remove newsfragments from this particular PR was correct. Because newsfragments should reference an issue they are closing. And this PR is not closing any issue
- My question was regarding the statement "I don't think newsfragments are used in OCA anymore". One would expect it look to like this: 
"Newsfragments are definitely not used in OCA anymore **because** <link_to_a_doc_pr_commit_or_any_other_proof>"

Actually newsfragmnets are being used by OCA. For example here is a [PR with newsfragments](https://github.com/OCA/rest-framework/pull/393/files) merged recently by @lmignon 
We just need to use them properly which wasn't done in this particular case 😄 

P.S. Don't worry too much your PR is approved already 😉

Reply:
@ivs-cetmix a search for "newsfragments" returns some 10 PRs opened in the last year, so I think @aleuffre's doubt on whether they are in use anymore was reasonable

Anyway, I'm still convinced these discussions can be more valuable in a dedicate issue in the maintainer repo rather than in random PRs :)